mkt/webapps/tests/test_models.py

```diff
@@ -155,8 +155,8 @@ def test_soft_deleted(self):
         # When an app is deleted its slugs and domain should get relinquished.
         post_mortem = Webapp.with_deleted.filter(id=app.id)
         eq_(post_mortem.count(), 1)
-        for attr in ('app_slug', 'app_domain'):
-            eq_(getattr(post_mortem[0], attr), None)
+        eq_(getattr(post_mortem[0], 'app_domain'), None)
+        eq_(getattr(post_mortem[0], 'app_slug'), '337141')
```

Comment:
We prevent developers setting all numeric slugs because there are a few methods (mainly API, iirc) that can work with an ID or a slug. 'deleted-337141' is a valid slug though, so could conflict :)